predictive_load.gemspec

```diff
@@ -11,5 +11,6 @@ Gem::Specification.new do |gem|
   gem.version       = '0.7.0'
 
   gem.add_runtime_dependency "activerecord", ">= 6.0", "< 7.1"
+  gem.add_development_dependency "pry"
```

Comment:
just add to Gemfile like all other dev deps

Reply:
You will find `byebug` is already listed in gemfiles/common.rb. Just change it to `gem 'pry-byebug'` and you should fine ✨

Reply:
Done and done.